README.md

```diff
@@ -1,2 +1,198 @@
+
+# 📖 TV SHOWS CAPSTONE <a name="about-project"></a>
+
+**TV SHOWS CAPSTONE** is a tool that helps movie fans see the latest movies. It simply lists the movies that have been released and allows you to add comments.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+</details>
+<details>
+  <summary>Javascript: ES6</summary>
+</details>
+<details>
+  <summary>CSS</summary>
+</details>
+<details>
+  <summary>Webpack</summary>
+</details>
+<details>
+  <summary>Jest</summary>
+</details>
+<details>
+  <summary>REST APIs</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **List structure layout**
+- **Fetch TV Shows from API**
+- **Display the fetched TV Shows**
+- **Like a TV Show(s)**
+- **Increment like counter and update view**
+- **Popup window to show more TV Show mdetails**
+- **Post comments to Involvement API**
+- **Fetch comments from API and display**
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Website Demo Link](https://egichuhis.github.io/tv-shows-capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Install NodeJS on your computer. You can download the installation file from the [NodeJS website](https://nodejs.org/en/download/).
+
+On Linux you can use your package manager, for example, on Ubuntu just run the following commands:
+
+```sh
+ sudo apt-get update
+ sudo apt-get install nodejs
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/egichuhis/tv-shows-capstone.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project:
+
+Open the Terminal / Command-Line utility in your computer / code editor.
+
+Run `npm run start`. A browser window will be launched automatically with your project
+
+### Run tests
+
+To run tests, run the following commands:
+
+Example command:
+
+```sh
+  npx hint .
+  npx stylelint "**/*.{css,scss}"
+  npx eslint .
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Edwin Gichuhi**
+
+- GitHub: [@githubhandle](https://github.com/egichuhis)
+- Twitter: [@twitterhandle](https://twitter.com/egichuhis)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/edwin-gichuhi/)
+
+👤 **Lucy W. Mwangi**
+
+- GitHub: [@lucy-sees](https://github.com/lucy-sees)
+- Twitter: [@\_The_LoneWolf](https://twitter.com/_The_LoneWolf)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/lucy-w-mwangi-83a31b11b/)
```

Comment:
- [ ] This link is currently broken as when I click on it, it displays an error 404 page, please kindly provide the appropriate link as it is unprofessional to leave broken links in your readme file 👍 ✨